docker/1.4.8/config/README.md

```diff
@@ -0,0 +1,4 @@
+# Configuration
+
+This directory is mounted as a volume for the docker container running Mosquitto.
+
```

Comment:
So, this single pull adds _two_ docker images? for two different out of date versions? With all the files duplicated?